lighthouse-core/computed/viewport-meta.js

```diff
@@ -11,10 +11,10 @@ const makeComputedArtifact = require('./computed-artifact.js');
 
 class ViewportMeta {
   /**
-   * @param {{MetaElements: LH.GathererArtifacts['MetaElements']}} artifacts
+   * @param {LH.GathererArtifacts['MetaElements']} MetaElements
    * @return {Promise<ViewportMetaResult>}
   */
-  static async compute_({MetaElements}) {
+  static async compute_(MetaElements) {
```

Comment:
a necessary refactor to ensure that they're still cached, also this seems like a bad idea to cache on the entire artifacts object anyway :)

Reply:
lol whoops